flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/config/handler/FlinkJedisClusterConfigHandler.java

```diff
@@ -44,8 +45,16 @@ public FlinkJedisConfigBase createFlinkJedisConfig(Map<String, String> propertie
             String[] arr = r.split(":");
             return new InetSocketAddress(arr[0].trim(), Integer.parseInt(arr[1].trim()));
         }).collect(Collectors.toSet());
-        return new FlinkJedisClusterConfig.Builder()
-                .setNodes(nodes).build();
+        String clusterPassword = properties.getOrDefault(REDIS_CLUSTER_PASSWORD, null);
+        if (clusterPassword != null && clusterPassword.trim().isEmpty()) {
```

Comment:
this if could be merged with the if on line 54 no?

if (clusterPassword != null && clusterPassword.trim().isEmpty()) {
       builder.setPassword(clusterPassword);
}

or directly
if (StringUtils.isNotBlank(clusterPassword))

Reply:
Thanks,  i optimized the code

Reply:
Thanks,  i optimized the code
if (StringUtils.isNotBlank(clusterPassword)) {
builder.setPassword(clusterPassword);
}